crates/workspace/Cargo.toml

```diff
@@ -35,6 +35,8 @@ call.workspace = true
 client.workspace = true
 clock.workspace = true
 collections.workspace = true
+dap-types = { git = "https://github.com/zed-industries/dap-types" }
```

Comment:
This can be removed right, because they are already inside the dap crate, and we export them there.